packages/coordinator/tests/services/translation/response/organization.spec.ts

```diff
@@ -83,65 +79,21 @@ describe.each([
 const authorLocations = createLocations(authorRepresentedOrganization)
 const performerLocations = createLocations(performerRepresentedOrganization)
 
-const authorHealthcareService = createHealthcareService(authorRepresentedOrganization, authorLocations)
-const performerHealthcareService = createHealthcareService(performerRepresentedOrganization, performerLocations)
-
 describe.each([
-  ["authorOrganization", authorLocations, authorHealthcareService, authorRepresentedOrganization],
-  ["performerOrganization", performerLocations, performerHealthcareService, performerRepresentedOrganization]
+  ["authorOrganization", authorLocations, authorRepresentedOrganization],
+  ["performerOrganization", performerLocations, performerRepresentedOrganization]
 ])(
-  "createLocations & createHealthcareService",
+  "createLocations",
   (
     organizationName: string,
     fhirLocations: Array<fhir.Location>,
-    fhirHealthcareService: fhir.HealthcareService,
     hl7Organization: hl7V3.Organization
   ) => {
-    test("%p has an identifier block with the correct value", () => {
-      expect(fhirHealthcareService.identifier).not.toBeUndefined()
-      const identifierValue = getIdentifierValueForSystem(
-        fhirHealthcareService.identifier,
-        "https://fhir.nhs.uk/Id/ods-organization-code",
-        "Organization.identifier"
-      )
-      expect(identifierValue).toBe(hl7Organization.id._attributes.extension)
-    })
-
-    test("%p has correct name value", () => {
-      expect(fhirHealthcareService.name).toBe(hl7Organization.name._text)
-    })
-
-    test("%p has correct telecom value", () => {
-      expect(fhirHealthcareService.telecom[0].system).toBe("phone")
-      expect(fhirHealthcareService.telecom[0].value).toBe(hl7Organization.telecom._attributes.value.split(":")[1])
-      expect(fhirHealthcareService.telecom[0].use).toBe("work")
-    })
-
-    test("%p has a reference to the location", () => {
-      expect(fhirHealthcareService.location[0].reference).toBe(`urn:uuid:${fhirLocations[0].id}`)
-    })
-
     test("%p has correct address value", () => {
```

Comment:
does this still work? I thought the location resources were only created in the secondary care format.

Reply:
It does still pass yeah, and the createLocations appears to still be used within the `createWithRefactorEnabled` function